src/balance/Balance.chain.ts

```diff
@@ -117,9 +119,17 @@ export async function listenToBalanceChanges(
 export async function makeTransfer(
   identity: Identity,
   accountAddressTo: IPublicIdentity['address'],
-  amount: BN
+  amount: BN,
+  exponent = -15
 ): Promise<SubmittableResult> {
   const blockchain = await getCached()
-  const transfer = blockchain.api.tx.balances.transfer(accountAddressTo, amount)
+  const cleanExponent =
+    (exponent >= 0 ? 1 : -1) * Math.floor(Math.abs(exponent))
+  const transfer = blockchain.api.tx.balances.transfer(
+    accountAddressTo,
+    cleanExponent === -15
+      ? amount
+      : BalanceUtils.convertToTxUnit(amount, cleanExponent)
+  )
```

Comment:
This might be worth waiting for the changes that the other PR will come out with some much needed refactoring of how we handle `tx`s.

Reply:
It looks very good though.